### PR TITLE
feat: Update App Exit Handling

### PR DESCRIPTION
**Pull request by jmeier204**

### CODE REVIEW

File: cypress/support/cypress-commands/commands.js [hunk 1]
```diff
@@ -1595,6 +1595,46 @@ Cypress.Commands.add('envConfigSetup', () => {
...
+      break;
+    case 'unloadApp':
+      exitMethod = CONSTANTS.REQUEST_OVERRIDE_CALLS.UNLOADAPP;
+      break;
+    case 'dismissApp':
+      exitMethod = CONSTANTS.REQUEST_OVERRIDE_CALLS.DISMISSAPP;
+      break;
+    default:
+      fireLog.info('Session for appId: ' + appId + ' will not be ended due to invalid exitType');
+      fireLog.info(CONSTANTS.CONFIG_IMPLEMENTATION_MISSING).then(() => {
```
Comment (jbigel) [inline cypress/support/cypress-commands/commands.js:1624]: Isn't there a fireLog function that will assert and log all in one?

File: defaultModule/requestModules/fcs.js [hunk 2]
```diff
@@ -132,21 +132,103 @@ function triggerEvent(key) {
...
+ **/
+function dismissApp(request) {
+  // Generic error message due to missing implementation.
+  const appId = request.params;
+  fireLog.info(
+    'App dismiss requires platform implementation. AppId: ' +
+      appId +
+      ' was not able to be dismissed.'
+  );
+  fireLog.info(CONSTANTS.CONFIG_IMPLEMENTATION_MISSING).then(() => {
```
Comment (jbigel) [inline defaultModule/requestModules/fcs.js:165]: same as above

Comment (jmeier204) [reply defaultModule/requestModules/fcs.js]: see above

**Review by jbigel — APPROVED** [review]

**Pull request merged**